api/api-pipeline-instance-v1/src/main/java/com/thoughtworks/go/apiv1/pipelineinstance/representers/MaterialRepresenter.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtworks.go.apiv1.pipelineinstance.representers;
+
+import com.thoughtworks.go.api.base.OutputWriter;
+import com.thoughtworks.go.domain.materials.Material;
+
+public class MaterialRepresenter {
+    public static void toJSON(OutputWriter outputWriter, Material material) {
+        outputWriter.add("id", material.getId())
```

Comment:
Current representation of material config is:

```json
{
  "id": 1,
  "name": "test-repo",
  "fingerprint": "de08b060ebfc1fb988b6683bf21",
  "type": "Git",
  "description": "URL: https://github.com/gocd/gocd, Branch: master"
}
```

does it make sense to have the following representation

```json
{
  name: "test-repo",
  fingerprint: "fksdhfksdg",
  type: "git",
  url: "https://******@github.com/gocd/gocd"
  branch: "master"
}
```